feat(routes): childRoutes are not required for root module (#657)
americanexpress/one-app: docs/api/modules/Routing.md

route, array of routes or a function which accepts the Redux store as the sole argument and returns
a single route or array of routes.

> `childRoutes` is optional for Root Module. `childRoutes` can be either `<Route>` or `<ModuleRoute>` with a `path`. If `childRoutes` is not defined, default path `/` is added to RootRoute prop in one app.

Reviewer: ```suggestion
`childRoutes` is optional for Root Module. `childRoutes` can be either `<Route>` or `<ModuleRoute>` with a `path`. If `childRoutes` is not defined, the default path `/` is added to the root route.
```
